### PR TITLE
Blog post: Static sites

### LINKED ISSUE #249: Blog post: Static sites @ Compiler
Blog post writer: Machiko Yasuda
Text: https://docs.google.com/document/d/1uPhtkd0fpx3DiTNaB-c8ti043r7y9hnVFhEorDgm7Xk/edit?tab=t.0

Headline: How Compiler approaches static sites
Subhead: Learn how the Engineering team combines older and reliable static site frameworks with new build tools and an open source approach to craft memorable and functional content-driven sites.

Social media blurb: Learn how the Engineering team combines older and reliable static site frameworks with new build tools and an open source approach to craft memorable and functional content-driven sites.

Art: 
Image

Alt: A map of California counties, labeled by how many transit providers in each county. A table of transit agency name, GTFS links and other data.

Image

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #249 

Blog post writer: Machiko Yasuda
Text: In this PR

Headline: How Compiler approaches static sites
Subhead: Learn how the Engineering team combines older and reliable static site frameworks with new build tools and an open source approach to craft memorable and functional content-driven sites.

Social media blurb: Learn how the Engineering team combines older and reliable static site frameworks with new build tools and an open source approach to craft memorable and functional content-driven sites.

### CODE REVIEW

**Comment by machikoyasuda** [comment]
This post is ready for editing. @laneymangan

File: src/_posts/2024-11-01-static-sites-at-compiler.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Vast improvements in CSS over the past few years, with features like [CSS variables](https://blog.getbootstrap.com/2022/05/16/using-bootstrap-css-vars/) - which are now being implemented in Bootstrap, have allowed us to even stick with CSS over pre-processors like Sass that might encourage us to write _more_ code in even more files.
+
+## Getting the whole team involved with GitHub and Netlify
+
+Writing a static site with Jekyll means that developers ultimately have to initiate and deploy the site, unlike a tool like WordPress or Webflow where the people writing the actual content can be in charge of publishing. With our stack, however, we use GitHub and Netlify to get designers, copy writers and stakeholders involved in more parts of the site creating process:
+
+- Anyone can create a GitHub ticket to request new content to be added, content to be edited.
+- GitHub issues can be customized to ensure developres are getting all the data, metadata they need from stakeholders to make the needed updates. Here's an example of a [issue template for a recent Cal-ITP project](https://github.com/cal-itp/calitp.org/issues/new?template=new_resource.yml).
```
Comment (angela-tran) [inline src/_posts/2024-11-01-static-sites-at-compiler.md:45]:
```suggestion
- GitHub issues can be customized to ensure developers are getting all the data, metadata they need from stakeholders to make the needed updates. Here's an example of a [issue template for a recent Cal-ITP project](https://github.com/cal-itp/calitp.org/issues/new?template=new_resource.yml).
```

**Comment by machikoyasuda** [comment]
@laneymangan @angela-tran I updated the copy with the latest from Google Docs. Thank you for all the edits.

I changed the date to December 3rd.... which, sorry, I realized, makes the preview link not work.

The best way to preview the code/text now is to look here: https://github.com/compilerla/compiler.la/blob/19f4959fb153e01fb0a096d54ef9b62c12348b0c/src/_posts/2024-11-01-static-sites-at-compiler.md

**Review by angela-tran — APPROVED** [review]

**Comment by angela-tran** [comment]
I just manually triggered a new deploy preview in Netlify for this PR so that we can preview the post. :eyes:

**Review by laneymangan — APPROVED** [review]

**Pull request merged**